torch/csrc/jit/codegen/cuda/fusion.h

```diff
@@ -165,7 +165,10 @@ class TORCH_CUDA_API Fusion final {
   std::unordered_set<Expr*> unordered_uses(Val* val) const;
 
   // Return the Expr that produces val
-  Expr* origin(const Val* val) const;
+  Expr* origin(Val* val) const;
+
+  // Return the Expr that produces val
+  const Expr* origin(const Val* val) const;
```

Comment:
1.  we could have a single overload: `Expr* origin(const Val* val) const`
 (we don't have deep constness in Fusion, so this is perfectly fine)

2. if we go down this route, please add a todo note to completely remove Fusion::origin()